fuel-tx/src/tests/valid_cases/transaction.rs

```diff
@@ -123,6 +123,56 @@ fn maturity() {
     assert_eq!(ValidityError::TransactionMaturity, err);
 }
 
+#[test]
+fn expiration() {
```

Comment:
I know you're just basing this on the old tests, but can you add the SUT and G/W/T to this test.

This is a good example of why we should include those. I'm having a hard time understanding what the two `assert`s are for, so this should _probably_ be 2 tests as well.

Reply:
Yes or at least we should have some comments throughout the test explaining what is being tested and what the expected result is.

Reply:
Yeah I just wanted to follow the previous "convention" but okey I will go for the new improved way. Done.